packages/pixeloven-generators/src/templates/Component/function.Component.tsx.hbs

```diff
@@ -0,0 +1,20 @@
+import React{{#if includeState}}, { useState }{{/if}} from "react";
+
+import "./{{componentName}}.scss";
+
+interface {{componentName}}Props {
+    exampleProp?: string;
+}
+
+function {{componentName}}(props: {{componentName}}Props) {
+    const { exampleProp } = props;
+    {{#if includeState}}
+    const [exampleState] = useState(0);
```

Comment:
Should we include the interface for state and apply to this?

Reply:
In this case it's inferred right? Looks like the only time we'd need it is if we have an initial value of `undefined`.

Reply:
So yes and no. Yes it will infer based on the defaults but if those values can be anything else then you might run into some type errors.